app/templates/sign-up.hbs

```diff
@@ -5,10 +5,11 @@
   <div class="row">
     <div class="well col-md-6 col-md-offset-3 sign-up-section">
       <span class="primary">Sign-up with your Github Account Today!</span>
-      <span class="muted-text">Something less important goes here.</span>
       <hr>
-      <button class="btn btn-turing btn-lg github-signup"><i class="fa fa-github-square fa-2x" aria-hidden="true"></i></button>
+      <a href="http://localhost:3000/start_auth" class="btn btn-turing btn-lg github-signup"><i class="fa fa-github-square fa-2x" aria-hidden="true"></i></a>
```

Comment:
I like that we have this working, next step is too set it up to redirect correctly on both localhost and production.